packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -122,6 +133,14 @@ export default class QuanticGeneratedAnswer extends LightningElement {
   ariaLiveMessage;
   /** @type {boolean} */
   hasInitializationError = false;
+  /** @type {boolean} */
+  _answerDisplayUpdated = false;
```

Comment:
Unless I'm mistaken, I don't see that this is used anywhere?

Reply:
You're correct this was a remnant of the previous way I was handling the collapse.

I've had a lot of trouble while writing the tests because the answer is never "streaming" when we write our e2e tests because we just mock the "final" message.

So anything that was based off of `isStreaming` wasn't working correctly but just because of our tests.